sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -964,6 +964,35 @@ multiple of 4 when `T` is `float`; where `T` is the type of the
 `joint_matrix` elements. When `T` is not `half` or `float` there are
 no restrictions to `stride`.
 
+==== AMD Matrix Cores Supported Combinations
+The complete set of matrix data types and dimenstions that are supported by
+the `ext_oneapi_hip` backend are represented in the following
+table. In this architecture's implementation, A and B matrices must have the same type. 
+Similarly, C and D matrices must share the same type.
+
+IMPORTANT: Currently, only one block AMD Matrix Core instructions in 
```

Comment:
What you refer to as "block" is not explained in this document, and most people would probably think that you mean `work_group` by "block" since in CUDA block == work_group. I suggest simply removing this sentence and similar references to "block".

Reply:
Thanks @JackAKirk 
Makes sense, the sentence is removed.